libraries/c_sdk/standard/ble/CMakeLists.txt

```diff
@@ -112,20 +92,19 @@ afr_module_dependencies(
     PUBLIC
         AFR::serializer
         AFR::ble
-	    AFR::core_mqtt
+        AFR::wifi
```

Comment:
Why is this dependency required? Would this dependency cause problems for the Nordic board that does not support this target?

Reply:
Nope wifi provisioning module for BLE also requires WiFi dependency and its not enabled for Nordic. This is not changed.